documents/policies/internal/Email Account.md

```diff
@@ -30,20 +30,21 @@ WCA Volunteers must not use their WCA email to:
 WCA Volunteers should not use their WCA email for personal reasons.
 
 ### Email Signature
-WCA Volunteers are encouraged to create a professional email signature. The email signatures should not contain logos or unnecessary textual references. Here is a sample acceptable email signature:  
-**Mike Miller**  
-*Member of the WCA Results Team*  
+WCA Volunteers are encouraged to create a professional email signature. The email signatures should not contain logos or unnecessary textual references. Here is a sample acceptable email signature:
+
+**Mike Miller**
+*Member of the WCA Results Team*
 *WCA Delegate for United States (New York)*
 
 ### Disciplinary Action
-WCA Volunteers who do not adhere to this policy may face disciplinary action (including dismissal from their volunteer role).
+WCA Volunteers who do not adhere to this policy may face disciplinary action up to and including dismissal from their Volunteer role.
```

Comment:
```suggestion
WCA Volunteers who do not adhere to this policy may face disciplinary action, including dismissal from their Volunteer role.
```
The previous edited changes the meaning slightly, as it now suggests that disciplinary action beyond such dismissal, e.g. a ban, is now not possible, whereas formerly it was not precluded.